.prettierignore

```diff
@@ -0,0 +1,4 @@
+/node_modules
+.env
+.md
```

Comment:
replace this with `**/*.md`
and same for json files (i.e. `**/*.json`)

this will make sure all markdown and json files are ignored (across all directories)

Reply:
you have to do `npx prettier --write .` again after this

Reply:
Done